sakura_core/CSortedTagJumpList.h

```diff
@@ -55,10 +55,10 @@ class CSortedTagJumpList {
 	int GetCapacity(void) const { return m_MAX_TAGJUMPLIST; }
 
 private:
-	TagJumpInfo*	m_pTagjump;	//!< タグジャンプ情報
+	TagJumpInfo*	m_pTagjump = nullptr;	//!< タグジャンプ情報
 	std::vector<std::wstring> m_baseDirArr;	//!< ベースディレクトリ情報
```

Comment:
指摘じゃないです。

```suggestion
	std::vector<std::wstring> m_baseDirArr{ L"" };	//!< ベースディレクトリ情報
```

こうしたら `m_baseDirArr[0]` に空白文字列が設定された size==1 の 可変長配列 を初期化できるはず。